AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/manage-team/manage-team.component.html

```diff
@@ -2,15 +2,108 @@
       </div>
     </div>
   </div>
+  <div *ngIf="displayMessage" class="govuk-!-font-weight-bold" [ngClass]="errorMessage ? 'vh-text-color-red' : ''">
+    <fa-icon [icon]="errorMessage ? faError : faExclamation"></fa-icon>
+    {{ message }}
+  </div>
+  <table class="govuk-table" data-module="moj-sortable-table" aria-describedby="justice user table">
+    <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header">Username</th>
+        <th scope="col" class="govuk-table__header">First name</th>
+        <th scope="col" class="govuk-table__header">Last name</th>
+        <th scope="col" class="govuk-table__header">Contact email</th>
+        <th scope="col" class="govuk-table__header">Role</th>
+        <th scope="col" class="govuk-table__header action-cell"></th>
+        <th scope="col" class="govuk-table__header action-cell"></th>
+      </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+      <tr class="govuk-table__row" *ngFor="let user of users">
+        <td class="govuk-table__cell">
+          <input
+            id="username_{{ user.id }}"
+            [disabled]="true"
+            type="text"
+            [(ngModel)]="user.username"
+            class="{{ user.id }}"
+            required="true"
+          />
+        </td>
+        <td class="govuk-table__cell">
+          <input
+            id="firstname_{{ user.id }}"
+            [disabled]="true"
+            type="text"
+            [(ngModel)]="user.first_name"
+            class="{{ user.id }}"
+            required="true"
+          />
+        </td>
+        <td class="govuk-table__cell">
+          <input
+            id="lastname_{{ user.id }}"
+            [disabled]="true"
+            type="text"
+            [(ngModel)]="user.lastname"
+            class="{{ user.id }}"
+            required="true"
+          />
+        </td>
+        <td class="govuk-table__cell">
+          <input
+            id="contact_email_{{ user.id }}"
+            [disabled]="true"
+            type="text"
+            [(ngModel)]="user.contact_email"
+            class="{{ user.id }}"
+            required="true"
+          />
+        </td>
+        <td class="govuk-table__cell">
+          <input
+            id="role_{{ user.id }}"
+            [disabled]="true"
+            type="text"
+            [(ngModel)]="user.user_role_name"
+            class="{{ user.id }}"
+            required="true"
+          />
+        </td>
+        <td class="govuk-table__cell">
+          <button
+            class="govuk-button govuk-!-margin-right-6"
+            data-module="govuk-button"
+            id="editButton_{{ user.id }}"
+            (click)="editUser(user.id)"
+          >
+            Edit
+          </button>
+        </td>
+        <td class="govuk-table__cell">
+          <button class="govuk-button govuk-!-margin-right-6 red-button" data-module="govuk-button" (click)="deleteUser(user.id)">
```

Comment:
think we can remove this click handler until we implement edit + delete

Reply:
done